chapters/1-reproducibility.tex

```diff
@@ -262,7 +273,10 @@ \subsection{Documenting data acquisition and analysis}
 % Tools for transparency: GitHub, OSF
 Email, however, is \textit{not} a documentation service, because communications are rarely well-ordered,\index{email}
 can be easily deleted, and are not available for future team members.
-There are various software solutions for building proper documentation over time.
+At the vey least, emails and other decision-making communications
+need to be archived and preserved (as, say, PDFs)
+so that they can be read easily by others in the future.
+There are also various software solutions for building proper documentation over time.
```

Comment:
give an example? or do we have any examples on the wiki?

Reply:
I thought we had just cut out mentions of things like GitHub issues and dropbox paper. I could add Asana or something? Alternatively we could add "project management" on Wiki but I don't think we have a ton to say about how to manage tasks internally?

Reply:
Ah, fair enough, we did cut those. Ok, i think it's fine to leave as it was.